source/Hazard.cpp

```diff
@@ -35,6 +35,9 @@ void Hazard::Load(const DataNode &node)
 		const string &key = child.Token(0);
 		if(key == "weapon")
 			LoadWeapon(child);
+		else if(key == "to" && child.Size() == 2 && child.Token(1) == "spawn")
+			// Handled by caller and stored in RandomEvent
+			continue;
```

Comment:
Is this necessary?

Reply:
Without it, there is a `child.PrintTrace("Skipping unrecognized attribute:");` warning.

Reply:
Are you absolutely sure? It doesn't seem to me like System::Load calls Hazard::Load.

Reply:
I tested it and you're right. It's gone now.